cmd/manager/main.go

```diff
@@ -64,6 +68,9 @@ func main() {
 			"Setting this value to 0 means no cache.")
 	flag.StringVar(&defaultCACertsPath, "default-ca-certs", "",
 		"The path to a PEM-encoded CA Certificate file to supply as default for OpenStack API requests.")
+	flag.StringVar(&watchedNamespaces, "namespaces", "",
```

Comment:
Please can we make this take a single namespace, but allow it to be specified multiple times? See https://stackoverflow.com/questions/28322997/how-to-get-a-list-of-values-into-a-flag-in-golang

Reply:
done!